playwright.config.ts

```
@@ -1,38 +1,34 @@
+  name: config.name,
   use: {
-    testItem: new TestItem(
-      template as TemplateType,
-      tssc.registry as ImageRegistryType,
-      tssc.git as GitType,
-      tssc.ci as CIType,
-      tssc.tpa || '',
-      tssc.acs || ''
-    )
-  },
+    testItem: config.testItem
```

ESLint

[error] Insert `,`

(prettier/prettier)